prometheus-authguard.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Prometheus Authguard Server Instance
```

Comment:
could you just remove the prometheus here and move the file to `authguard.service`? although prometheus was the initial use-case, I'd prefer to have the repo a little more generalized.